recipes/funcsigs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
```

Comment:
Probably would require the build customization stuff, but yes that seems possible.